queries/liquidsoap/highlights.scm

```diff
@@ -0,0 +1,83 @@
+(op) @operator
+"..." @operator
```

Comment:
Not yet ig
```suggestion
[
  (op)
  "..."
] @operator
```

And why is `...` so special here. Can't the parser make it so it's part of `(op)`

Reply:
Indeed. Just pushed a change making it an `op`. It works great, `...` in types is not recognized as `@operator` now.